.github/workflows/release_draft.yml

```diff
@@ -36,4 +36,46 @@ jobs:
           git config --global user.email "${{ github.actor }}@users.noreply.github.com"
 
       - name: checkout to latest minor release branch
-        run: git checkout release/${{ inputs.releaseVersion }}
+        run: |
+          git fetch origin release/${{ inputs.releaseVersion }}
+          git checkout release/${{ inputs.releaseVersion }}
+          git status
+
+      - name: bump patch version and commit
+        run: |
+          new_version=$(./entrypoint.sh bump_version commit patch)
+          echo $new_version
+          echo "NEW_PATCH_VERSION=$new_version" >> $GITHUB_ENV
+          git push
+
+      - name: Tag patch commit and push
+        run: |
+          git tag -f -a ${{ env.NEW_PATCH_VERSION }} -m "Release ${{ env.NEW_PATCH_VERSION }}"
+          git push origin ${{ env.NEW_PATCH_VERSION }}
+
+      - name: Create draft release
+        run: gh release create ${{ env.NEW_PATCH_VERSION }} --draft --title "Release ${{ env.NEW_PATCH_VERSION }}" --generate-notes
```

Comment:
can you add `--notes-start-tag` ? if we trust the auto version, we never have the right one. Or, they're is maybe some cleaning in the tags required. wdyt?

Reply:
I added this option because you're right, the auto version is not working properly for our repo. I am not sure why.

Those tags might be what's causing the issue, maybe...:
- 2.124.0rc0
- 2.124.0prerelease
- 2.124.0-prerelease